go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -1113,6 +1123,52 @@ func setBridgeOfPorts(bridge *bridgeConfiguration) error {
 	return nil
 }
 
+// initSvcViaMgmPortRoutingRules creates the svc2managementport routing table, routes and rules
+// that let's us forward service traffic to ovn-k8s-mp0 as opposed to the default route towards breth0
+func initSvcViaMgmPortRoutingRules(hostSubnets []*net.IPNet) error {
+	// create ovnkubeSvcViaMgmPortRT and service route towards ovn-k8s-mp0
+	for _, hostSubnet := range hostSubnets {
+		isIPv6 := utilnet.IsIPv6CIDR(hostSubnet)
+		gatewayIP := util.GetNodeGatewayIfAddr(hostSubnet).IP.String()
+		for _, svcCIDR := range config.Kubernetes.ServiceCIDRs {
+			if isIPv6 == utilnet.IsIPv6CIDR(svcCIDR) {
+				if stdout, stderr, err := util.RunIP("route", "replace", "table", ovnkubeSvcViaMgmPortRT, svcCIDR.String(), "via", gatewayIP, "dev", types.K8sMgmtIntfName); err != nil {
+					klog.Errorf("Error adding routing table entry into custom routing table: %s: stdout: %s, stderr: %s, err: %v", ovnkubeSvcViaMgmPortRT, stdout, stderr, err)
```

Comment:
why is this only a log instead of returning an error?

Reply:
ah good point, no idea why I didn't return this. let me fix this.